llvm/lib/Target/X86/CMakeLists.txt

```diff
@@ -1,126 +1,113 @@
 add_llvm_component_group(X86 HAS_JIT)
 
-set(LLVM_TARGET_DEFINITIONS X86.td)
+    set(LLVM_TARGET_DEFINITIONS X86.td)
```

Comment:
It looks like you tried to autoformat this? The changes here fail the build on premerge.